software/nosql/pom.xml

```diff
@@ -295,4 +295,26 @@
         </plugins>
       </pluginManagement>
     </build>
+
+  <profiles>
+    <profile>
+      <id>defaultProfile</id>
+      <activation>
+	<property><name>!brooklyn.cloudtest</name></property>
+      </activation>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-surefire-plugin</artifactId>
+            <configuration>
+              <excludes>
+                <exclude>**/CassandraFabricTest.java</exclude>
```

Comment:
Fair enough. I assume however this is not a blocker for merging the PR, right? We can take a look later (me or somebody more experienced in that particular area).

From what I recall the issue was related to host seeds for the cassandra cluster when running on just localhost. After all the refactoring it looks like that hack is no longer necessary. Will add a commit shortly.